pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -34,11 +34,18 @@ import (
 
 var bridgeFakeIP = "169.254.75.86/32"
 
+// DefaultProtocol is the default port protocol
+const DefaultProtocol string = "TCP"
+
+// DefaultVMCIDR is the default CIDR for vm network
+const DefaultVMCIDR = "10.0.2.0/24"
+
 type BindMechanism interface {
 	discoverPodNetworkInterface() error
 	preparePodNetworkInterfaces() error
-	startDHCPServer()
-	decorateInterfaceConfig(ifconf *api.Interface)
+	libvirtConfig() error
```

Comment:
I'd prefer not to mention `libvirt` specifically, rather more like decorateConfig. To me `decorate` makes it more generic, rather than `set` or `update`

Reply:
No problem will be change to decorateConfig